README.md

```diff
@@ -53,6 +53,7 @@ Name | Example | Description |
 
 Name | Example | Description |
 ---- | ------- | ----------- |
+`API_CONFIG_FILE` | custom-env.list | Specifies the path from which to load the config file. Environment variables take precedence over config file entries.
```

Comment:
should specify that this must specified in the environment, not in `env.list` like the other environment variables